pkg/operator/controller/status/controller.go

```diff
@@ -202,10 +206,34 @@ func initializeClusterOperator(co *configv1.ClusterOperator) {
 }
 
 type operatorState struct {
+	// haveNamespace indicates whether the operand namespace exists.
 	haveNamespace bool
-	namespace     corev1.Namespace
-	haveDNS       bool
-	dns           operatorv1.DNS
+
+	// namespace is the operand namespace.
+	namespace corev1.Namespace
+
+	// haveDNS indicates whether the "default" dnses.openshift.operator.io
+	// CRD exists.
```

Comment:
```suggestion
	// CR exists.
```
total nit, don't feel obligated to fix this.

Reply:
Sorry, couldn't help feeling obligated to fix "CRD" here and above, as well as the API group name.